ibridges/__main__.py

```diff
@@ -103,8 +105,7 @@ def main() -> None:
         print(f"Invalid subcommand ({subcommand}). For help see ibridges --help")
         sys.exit(1)
 
-
-def _set_ienv_path(ienv_path: Union[None, str, Path]):
+def _get_ibridges_conf(ienv_path):
```

Comment:
I find the variable and function nameing a bit confusing. `ienv_path` points to the irods environment file and the function returns the .ibridges/ibridges_cli.json ...